api/v1beta1/disruption_webhook.go

```diff
@@ -44,6 +44,9 @@ var handlerEnabled bool
 var defaultDuration time.Duration
 var cloudServicesProvidersManager *cloudservice.CloudServicesProvidersManager
 var chaosNamespace string
+var safemodeEnvironment string
+
+const safeModeEnvironmentAnnotation = "chaos.datadoghq.com/environment"
```

Comment:
I'm sorry to be triggered by that but one variable is `safemode` and the other is `safeMode` 😭 

```suggestion
const safemodeEnvironmentAnnotation = "chaos.datadoghq.com/environment"
```